scripts/deploy_example_gcp

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPTS_DIR/common"
+
+"$SCRIPTS_DIR/build_example" "hello_world"
+
+bazel_build_flags+=(
+    '--config=enc-sim'
+)
+
+# Prerequisites:
+# - Google Cloud SDK
+# - Environment variable GCP_PROJECT_ID: id of the GCP project to use.
```

Comment:
So which project should we use? `oak-ci`, or something else? :) Maybe set a default somewhere anyways, it's not like we are going to switch projects very often for now.

Reply:
Added a suggestion in the comment (oak-vm-testing). The same environment variable is used by the Packer configuration, and I don't think I should be setting environment variables in the script.